tests/integration/testlib/statelib.py

```diff
@@ -229,3 +229,10 @@ def _is_ipv6_link_local(ip, prefix):
     The IPv6 link local address range is fe80::/10.
     """
     return ip[:3] in ['fe8', 'fe9', 'fea', 'feb'] and prefix >= 10
+
+
+def get_all_macs(state):
```

Comment:
It might simplify the code with multiple states if this accepts multiple states (`def get_all_macs(*state): ...`)

Reply:
Also after looking at it for too long, the `all` can probably be removed, since `get_macs(*state)` is descriptive enough.